src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/arithmetic.txt

```diff
@@ -100,15 +100,9 @@ true
 >> 5 < 5
 false
 
->> 5 < 7
```

Comment:
.txt parser now throws an error on duplicate cases within a single file (this is repeated on line 94)